### PR TITLE
Remove social icons from footer

### PR DESCRIPTION
**Pull request by gxrneyme**
I spoke to @kailan today about the fact that the social icons linked to broken URLs, and he told me it was a known issue. I offered to send a PR across with these fixed, so here we are. :tada:

### CODE REVIEW

File: app/views/partials/_footer.haml [hunk 1]
```diff
@@ -5,16 +5,6 @@
...
       = link_to "Kailan Blanks", "https://github.com/kailan/"
-    %ul.social
-      %li.text-primary
-        %a{:href => "https://twitter.com/SyoStudentGov"}
-          = fa_icon 'twitter'
-      %li
-        %a{:href => "https://www.facebook.com/syosseths"}
-          = fa_icon 'facebook'
-      %li
-        %a{:href => "https://github.com/Syosset/syosset"}
```
Comment (kailan) [inline app/views/partials/_footer.haml]: Could we keep the GitHub link?

Comment (gxrneyme) [reply app/views/partials/_footer.haml]: Sounds good, will have to add the CSS back though. Watch this space.

**Comment by gxrneyme** [comment]
Just updated it now — let me know if it looks okay to you.

**Comment by gxrneyme** [comment]
Nice one! Cheers, @kailan — have set up the repo on my MacBook so will look into some of the other issues at some point.

**Comment by gxrneyme** [comment]
Nice one, thanks @kailan and @neiljohari.

**Comment by neiljohari** [comment]
@gxrneyme Thanks for your contribution, it’s very appreciated!

**Comment by gxrneyme** [comment]
@neiljohari Pleasure as always. Set the app up on my MB, so will look at some other issues at some point.

**Pull request merged**